internal/event/event.go

```diff
@@ -34,9 +34,10 @@ type Event struct {
 }
 
 const (
-	TypeState           = "state"
-	TypeAcknowledgement = "acknowledgement"
-	TypeInternal        = "internal"
+	TypeState                 = "state"
+	TypeAcknowledgement       = "acknowledgement"
+	TypeReevaluationScheduled = "reevaluation-scheduled"
+	TypeReevaluationStartup   = "reevaluation-startup"
```

Comment:
Now that I think of this, how much of a different event is this? I mean currently, any follow-up action is still due to the incident reaching a certain age, this simply additionally says that the processing might have been delayed due to the daemon not running at the time it happened.

@nilmerg You opened the corresponding issue, so that probably means that you'd want to display this differently. Is that correct and how would you display it?

(Something I've thought about in the past and that could serve as an alternative here: store two timestamps with each event: when it happened and when it was processed, that could then be used to show a generic "processing was delayed" indicator if there's a significant difference.)

Reply:
I'd have no objections about both using the same type. We'd display an icon and short textual phrase, in addition to the event's message. (e.g. :hourglass: Incident reached age)

Reply:
I have pushed a new version, effectively renaming `internal` to `reevaluation`. If future _internal_ events emerge, we could then introduce further event types.

Reply:
Is this now final? Can Web rely on these identifiers now?

Reply:
No, at least I'm not yet convinced enough that `reevaluation` isn't just too generic as well. Intuitively, I'd have named it more like `incident_age`, but I'm als not sure if this isn't too specific and how well we can isolate the actual reason (once there are more) that happened while the daemon was stopped and thus caused some action to happen right after daemon startup.

I'd give it a try to stay as specific as we can, worst case, we could still add an generic "we reevaluated the rule, now it matched, we don't really know the event that caused this" event type in the future, but at the same time, this sounds like something that should be avoided.

Reply:
No other comments/opinions/feedback on this? :sweat_smile: 

> We'd display an icon and short textual phrase, in addition to the event's message. (e.g. ⌛ Incident reached age)

If that's the stuff you want to display for that event type, it sounds more like you'd also expect something more like `incident_age`, at least "⌛ Incident reached age" wouldn't really fit for `reevaluation`.

Reply:
I discussed this with @nilmerg yesterday and it looks like there was some confusion due to the following messages being very different:
https://github.com/Icinga/icinga-notifications/blob/62e47fba02c7b19af7230b742bafc387ba855f9f/internal/incident/incidents.go#L129-L133
https://github.com/Icinga/icinga-notifications/blob/62e47fba02c7b19af7230b742bafc387ba855f9f/internal/incident/incident.go#L457-L461

However, in the end both are supposed to say something very similar: some trigger point is now or in the past. Thus, the suggestion would be to use `incident_age` and use more similar messages while still clearly pointing out that the event was delayed, something like "Incident reached age 3h14m (daemon was restarted)" (i.e. 3h14m being the current age, not the trigger time, so that event might trigger escalations for `incident_age>=2h` and `incident_age>=3h`)

Reply:
Please take another look.